src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball() {
+    }
```

Comment:
The default constructor does not initialize the `color` and `number` fields. If these fields are essential for a `Ball` object, consider removing this constructor or initializing the fields with default values.